editor/src/uuiui/radix-components.tsx

```diff
@@ -10,7 +10,9 @@ import { colorTheme, UtopiaStyles, UtopiaTheme } from './styles/theme'
 import { Icons, SmallerIcons } from './icons'
 import { when } from '../utils/react-conditionals'
 import { Icn, type IcnProps } from './icn'
-import { borderRadius } from './styles/utopitrons'
+import { forceNotNull } from '../core/shared/optional-utils'
+
+export const RadixComponentsPortalId = 'radix-components-portal'
```

Comment:
Maybe worth adding a comment here (on in the `.html` file?) to make sure if one is changed the other one changes too?

Reply:
Fixed in 045fe83f1.